tests/Areas/Workbooks/UnitTests/DeleteWorkbooksCommandTests.cs

```diff
@@ -0,0 +1,386 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.Text.Json;
+using Azure.Core;
+using AzureMcp.Areas.Workbooks.Commands.Workbook;
+using AzureMcp.Areas.Workbooks.Services;
+using AzureMcp.Models.Command;
+using AzureMcp.Options;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using Xunit;
+
+namespace AzureMcp.Tests.Areas.Workbooks.UnitTests;
+
+[Trait("Area", "Workbooks")]
+public class DeleteWorkbooksCommandTests
+{
+    private readonly IServiceProvider _serviceProvider;
+    private readonly IWorkbooksService _service;
+    private readonly ILogger<DeleteWorkbooksCommand> _logger;
+    private readonly DeleteWorkbooksCommand _command;
+
+    public DeleteWorkbooksCommandTests()
+    {
+        _service = Substitute.For<IWorkbooksService>();
+        _logger = Substitute.For<ILogger<DeleteWorkbooksCommand>>();
+
+        var collection = new ServiceCollection();
+        collection.AddSingleton(_service);
+        _serviceProvider = collection.BuildServiceProvider();
+
+        _command = new(_logger);
+    }
+
+    [Fact]
+    public void Constructor_InitializesCommandCorrectly()
+    {
+        var command = _command.GetCommand();
+        Assert.Equal("delete", command.Name);
+        Assert.NotNull(command.Description);
+        Assert.NotEmpty(command.Description);
+        Assert.Contains("workbook", command.Description, StringComparison.OrdinalIgnoreCase);
+        Assert.Contains("delete", command.Description, StringComparison.OrdinalIgnoreCase);
+        Assert.Contains("cannot be undone", command.Description, StringComparison.OrdinalIgnoreCase);
```

Comment:
@matteing why does this look for "cannot be undone" in the command?  delete of workbooks *can* be undone with recycle bin, so not sure why a test would be looking for this text in the command?